### PR TITLE
Liquibase

### PR DESCRIPTION
**Pull request by OlgaEvdokimova**
Create DML Hibernate connection after inizialization tables with liquibase

### CODE REVIEW

**Review by ShirshovAlex — DISMISSED** [review]

**Review by velichko1882 — DISMISSED** [review]

**Review by Maksim-maevsky — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -16,10 +17,11 @@
         <maven.compiler.target>11</maven.compiler.target>
         <postgresql.version>42.2.19</postgresql.version>
         <liquibase.version>4.3.2</liquibase.version>
+        <spring.data.jpa>2.4.4</spring.data.jpa>
+        <liquibase.version>4.3.2</liquibase.version>
```
Comment (kzaretskaya) [inline pom.xml]: excessive property. it's already added above

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -2,12 +2,15 @@ spring:
...
   datasource:
     driver-class-name: org.postgresql.Driver
-    url: jdbc:postgresql://localhost:5432/internship123
-    password: "0311"
+    url: jdbc:postgresql://localhost:5432/internship
+    password: "mama071991"
     username: postgres
+    initialization-mode: always
+    data:
+      - 'classpath:import.sql'
```
Comment (kzaretskaya) [inline src/main/resources/application.yaml]: is it supported to plug in 2 files? (each for appropriate entity)

File: pom.xml (hunk 1 above)
Comment (OlgaEvdokimova) [reply pom.xml]: done

File: src/main/resources/application.yaml (hunk 2 above)
Comment (OlgaEvdokimova) [reply src/main/resources/application.yaml]: File import.sql has test data which fill out two tables : form and internship

Comment (kzaretskaya) [reply src/main/resources/application.yaml]: I understand it. Is it possible to split this file and plug in to application.yaml separately?

Comment (OlgaEvdokimova) [reply src/main/resources/application.yaml]: Yes, I checked and got two filled tables out. Should I leave two separate files?

Comment (kzaretskaya) [reply src/main/resources/application.yaml]: I suggest to split it. It will be more readable.

Comment (OlgaEvdokimova) [reply src/main/resources/application.yaml]: done

**Review by viktarkolbik — APPROVED** [review]

**Review by kzaretskaya — APPROVED** [review]

**Pull request merged**